integration/tests/cook/test_basic.py

```diff
@@ -337,3 +340,42 @@ def test_application_field(self):
         # Should require application version
         _, resp = self.submit_job(application={'name': 'foo-app'})
         self.assertEqual(resp.status_code, 400)
+
+    def test_allow_partial(self):
```

Comment:
I should have been clearer. I meant to say the non-partial variant that loads multiple jobs. The one in this test handles only a single job.